frontend/package.json

```diff
@@ -68,6 +68,7 @@
     "build": "npm run lint && EXTEND_ESLINT=true react-scripts build",
     "build:protos": "cd .. && node frontend/scripts/gen_grpc_web_protos.js",
     "build:replace": "node scripts/replace_protos.js",
+    "build:k8s-platform-spec": "bash ./scripts/k8splatformspec.sh",
```

Comment:
I would suggest to cascade the hierarchy here: `build:platform-spec:kubernetes-platform`